gradle/verification-metadata.xml

```diff
@@ -860,9 +860,9 @@
             <pgp value="7186BBF993566D8C2F4F7ED7D945E643368FEF62"/>
          </artifact>
       </component>
-      <component group="io.github.gradle" name="gradle-enterprise-conventions-plugin" version="0.7.6">
-         <artifact name="gradle-enterprise-conventions-plugin-0.7.6.jar">
-            <sha256 value="b3ff4bba6c5ac6e50e7225c7d1f69c0a9bc2ae9f3ce2bfacdc3f1b3999c2e758" origin="Verified" reason="Artifact is not signed"/>
+      <component group="io.github.gradle" name="gradle-enterprise-conventions-plugin" version="0.10.0">
+         <artifact name="gradle-enterprise-conventions-plugin-0.10.0.jar">
+            <sha256 value="616a3b59e603d1e22f6da9cae020fb3d007163cfa3fa85b57fa0eac394e46a95" origin="Generated by Gradle"/>
```

Comment:
❌ `"Generated by Gradle"` needs to be handled, see the comment at the top of this file